app/views/users/index.html.erb

```diff
@@ -1,10 +1,10 @@
-<h1>User Details</h1>
+<h1>List of users</h1>
 <% @users.each do |user| %>
```

Comment:
[OPTIONAL] It is a good practice to use rails partials in scenarios like this. Kindly read up on it.

Reply:
ok, ill look unto that